net/file_server.ts

```diff
@@ -12,6 +12,81 @@ import {
   Response
 } from "./http.ts";
 import { cwd, DenoError, ErrorKind, args, stat, readDir, open } from "deno";
+import { extname } from "https://deno.land/x/path/index.ts";
+
+// based on https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types
+const extensionsMap = new Map([
```

Comment:
Probably we could move this to another file (or even to a separate module)? It is possible that later other modules might also need such mappings...

Reply:
Having it in a JSON file would be good